pnpm support

[pnpm](https://github.com/pnpm/pnpm/) is a wonderful replacement for npm, but it isnt working with `netlify-local`.

```
pnpm -g i netlify-local
Resolving: total 269, reused 205, downloaded 00 0wnloaded 0
Packages: +1
++-
Resolving: total 379, reused 377, downloaded 2, done

/home/jayvdb/vs/node/8.13.0/pnpm-global/1:
+ netlify-local 1.6.0

jayvdb@linux-lwww:~/projects/viperdev/template/ionic> netlify-local serve
module.js:550
    throw err;
    ^

Error: Cannot find module 'serve-static'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/jayvdb/vs/node/8.13.0/pnpm-global/1/node_modules/.registry.npmjs.org/netlify-local/1.6.0/node_modules/netlify-local/.dist/server.js:52:19)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
```

After a little patch, and a little fiddle, 

```console
> pnpm run test

> netlify-local@1.6.0 pretest /home/jayvdb/projects/netlify/netlify-local
> pnpm run build


> netlify-local@1.6.0 build /home/jayvdb/projects/netlify/netlify-local
> tsc --project ./tsconfig.json


> netlify-local@1.6.0 test /home/jayvdb/projects/netlify/netlify-local
> ts-mocha --project ./test/tsconfig.json test/**/*.ts



  Config
    parseNetlifyConfig
      ✓ should throw when not found
      ✓ should correctly parse build
      ✓ should correctly override build with context
    parseWebpackConfig
      ✓ should throw when not found
      ✓ should correctly import webpack config

  Server
    lifecycle
      ✓ should listen and close
      ✓ should add redirect routes
      ✓ should add redirect header routes
      ✓ should add header routes
      ✓ should add lambda route
      ✓ should correctly format lambda request
      ✓ should correctly format lambda context


  12 passing (202ms)

```

--- a/package.json
+++ b/package.json
@@ -27,6 +27,7 @@
     "@types/jsonwebtoken": "^8.3.0",
     "@types/mocha": "^5.2.5",
     "@types/node": "^10.11.7",
+    "@types/serve-static": "^1.13.2",
     "@types/webpack": "^4.4.16",
     "assert": "^1.4.1",
     "mocha": "^5.2.0",
@@ -44,6 +45,7 @@
     "jsonwebtoken": "^8.3.0",
     "toml": "^2.3.3",
     "url-pattern": "^1.0.3",
+    "serve-static": "^1.13.2",
     "webpack": "^4.20.2"
   }
 }

